jni/utilities/tablepool.h

```diff
@@ -32,26 +33,27 @@ class TablePool
 
         // retrieves the pooled table for given waveformType
 
-        static WaveTable* getTable( int waveformType );
+        static WaveTable* getTable( std::string tableId );
 
```

Comment:
TablePool now registers Tables by a unique string identifier. This allows us to register multiple WaveTables per type (e.g. WaveForms::CUSTOM)